src/main/java/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,45 @@
+package service.impl;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import model.FruitTransaction;
+import service.FileReader;
+
+public class FileReaderImpl implements FileReader {
+    private final String filePath;
+
+    public FileReaderImpl(String filePath) {
+        this.filePath = filePath;
+    }
+
```

Comment:
```suggestion
    public FileReaderImpl() {
    }
```
use file path as a method parameter